train.py

```diff
@@ -1395,37 +1425,6 @@ def update_training_scores_history(perf_scores_history, model, top1, top5, mAP,
                                score.epoch)
 
 
```

Comment:
Have we removed evaluation option related doc/s as well

Reply:
From the user experience perspective, nothing has been changed by removing the evaluate_model function. Previously, we were calling evaluate_model which was calling the test function. Now, we are calling the test function directly. The args.quantize_eval function was never utilized from our side, and when we transferred the repo to Pytorch 2 the option was removed from the args.

Reply:
I see, thanks a lot